src/components/attendee/WebcamGrid/WebcamGrid.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { faTimesCircle } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useVideoComms } from "components/attendee/VideoComms/hooks";
+import { VideoCommsParticipant } from "components/attendee/VideoComms/VideoCommsParticipant";
+
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { useUserId } from "hooks/user/useUserId";
+
+import styles from "./WebcamGrid.module.scss";
+
+interface TableGridProps {
+  space: WithId<AnyVenue>;
```

Comment:
I didn't notice anything other than the `id` being used, so this might relax dependencies on parents that include `WebcamGrid`

```suggestion
  spaceId: SpaceId; 
```